aiomcache/pool.py

```diff
@@ -39,29 +37,20 @@ def acquire(self):
 
         :return: ``tuple`` (reader, writer)
         """
-        while self._size < self._minsize:
+        while self.size() == 0 or self.size() < self._minsize:
```

Comment:
Because `_minsize` can be 0 and then pool would end up being blocked because no connections exist